README.md

```diff
@@ -32,12 +32,18 @@ The implementation of GA4 and the Crown Logo update are contained across multipl
 
 Several environment variables are declared within the CRIs and then used within the shared code of this repository:
 
-- `GA4_DISABLED` - Feature flag to disable GA4, defaulted to `false`
-- `UA_DISABLED` - Feature flag to disable UA, defaulted to `true`
+- `GA4_ENABLED` - Feature flag to enable GA4, defaulted to `false`
+- `UA_ENABLED` - Feature flag to enable UA, defaulted to `false`
 - `UA_CONTAINER_ID` - Container ID for Universal Analytics, required for UA to work correctly. Default value is `GTM-TK92W68`
 - `GA4_CONTAINER_ID` - Container ID for GA4, required for analytics to work correctly. Default value is `GTM-KD86CMZ`
 - `ANALYTICS_COOKIE_DOMAIN` - Cookie domain to persist values throughout the different sections of the OneLogin journey. Default value is `localhost`
 - `ANALYTICS_DATA_SENSITIVE` - Redacts all form response data, defaulted to `true`. Only to be set to `false` if a journey section contains no PII in none text based form controls
+- `GA4_PAGE_VIEW_ENABLED` - Feature flag to enable GA4 page view tracking, defaulted to `true`
```

Comment:
[Node environment variables](https://nodejs.org/docs/latest-v20.x/api/process.html#process_process_env) are all strings. This is because they are based off [environ](https://man7.org/linux/man-pages/man7/environ.7.html) which uses an array of strings internally.

This means that these values would be more corrected documented as `"true"` not `true`, and that this should be reflected in the implementations downstream. The confusion of over this node feature has been reflected by two different implementation styles ([strings in address-front](https://github.com/govuk-one-login/ipv-cri-cic-front/blob/64492abdd4e24c792666f7b8aea21343f52b70c3/src/lib/config.js#L28) and [booleans in cic-front](https://github.com/govuk-one-login/ipv-cri-cic-front/blob/64492abdd4e24c792666f7b8aea21343f52b70c3/src/lib/config.js#L28))